instaclustr/resource_cluster.go

```diff
@@ -668,14 +676,13 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error{
 		}
 	}
 	rackCount := len(rackList)
-	nodesPerRack := nodeCount/rackCount
+	nodesPerRack := nodeCount / rackCount
```

Comment:
possible div by 0 here ?

Reply:
Don't think that should be possible. As long as the api gives a rack associated with any node it'll be >0. (also, I just auto-changed spacing, so not really *my* code)

Reply:
think we should guard against that (im imagining a scenario where the nodes are being provisioned for example) , a simple if rackCount >0 otherwise nodesPerRack = 0 should sort it ?